tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpNoHearbeatsTest.java

```diff
@@ -80,4 +88,68 @@ public void inspectOpenedResource(Connection connection) {
       connection.close();
    }
 
+   private static final String QUEUE_NAME = "queue://testHeartless";
```

Comment:
Why does a test named "testCloseConsumerOnConnectionReset" use a queue named "queue://testHeartless", when "testHeartless" is the name of a different test in the class? Does it need a "queue://" prefix? 
I'd just use getTestName() inside the test get a helpful value.

Reply:
I made a mistake. I created a static property because this needs a Spawned VM. Getting it from the static property seemed easier.

Do you really care about this? if so we can change the test.